packages/travix-ui-kit/components/slidingPanel/slidingPanelHeader/slidingPanelHeader.js

```diff
@@ -3,20 +3,54 @@ import React from 'react';
 import classnames from 'classnames';
 import { getDataAttributes } from '../../_helpers';
 
-const renderDefaultLeftBlock = (backButtonLabel, onBackButtonClick) => (
-  <button
-    className="ui-sliding-panel-header__left-block-back"
-    onClick={onBackButtonClick}
-  >
-    <span className="ui-sliding-panel-header__left-block-back-icon" />
-
-    { backButtonLabel && (
-      <span className="ui-sliding-panel-header__left-block-back-text">
-        {backButtonLabel}
-      </span>
-    ) }
-  </button>
-);
+const renderLeftBlock = (
+  backButtonLabel,
+  leftBlock,
+  onBackButtonClick,
+  useDefaultLeftBlock,
+) => {
+  const defaultProps = {
+    className: "ui-sliding-panel-header__left-block-back",
+    onClick: onBackButtonClick,
+  };
+
+  if (leftBlock) {
+    return leftBlock(defaultProps);
```

Comment:
I think we should document somehow what props are forwarded to `leftBlock` and `rightBlock`.

Reply:
documented this props above prop types validation. Like that - https://github.com/Travix-International/ui/pull/13/commits/e210bb2ed91a4edb44a1c4621f8767c25fa76eb1#diff-ffc169c7ba9e533e8c6d8744cff8d248R120
If you have better idea, just let me know